packages/vetrina/src/Vetrina/Products/Premium.purs

```diff
@@ -16,16 +16,16 @@ hblPremium =
       DOM.div_
         [ DOM.text "Kvalitetsjournalistik när, var och hur du vill."
         , DOM.br {}
-        , DOM.text "Läs Hufvudstadsbladet för 1€ i en månad, därefter 9,99€ / månad tills vidare. Avsluta när du vill."
+        , DOM.text "Läs Hufvudstadsbladet gratis i en månad, därefter 9,99€ / månad tills vidare. Avsluta när du vill."
         ]
   , descriptionPurchaseCompleted: DOM.text "Du kan nu läsa Premiumartiklar på HBL.fi."
   , campaign: Just
       { no: 4071
```

Comment:
This needs change too. 4349

Reply:
I hate to see that I failed to add that, sorry!